ui/base-tool/src/main/java/org/cru/godtools/base/tool/analytics/model/OpenToolsAnalyticsActionEvent.kt

```diff
@@ -6,3 +6,4 @@ import org.cru.godtools.analytics.model.AnalyticsSystem
 object ToolOpenedAnalyticsActionEvent : AnalyticsActionEvent("tool-opened", system = AnalyticsSystem.APPSFLYER)
 object FirstToolOpenedAnalyticsActionEvent :
     AnalyticsActionEvent("first-tool-opened", system = AnalyticsSystem.APPSFLYER)
+object ToolOpenedViaShortcutAnalyticsActionEvent : AnalyticsActionEvent("tool-opened-shortcut", system = AnalyticsSystem.APPSFLYER)
```

Comment:
the `system = APPSFLYER` parameter limits this event to appsflyer only. for now let's change it to be Adobe only.

We will revisit it when we do the adobe -> google 360 migration in a couple weeks

Reply:
also, let's move this event object to it's own source file

Reply:
Okay sounds good.